src/App.jsx

```diff
@@ -0,0 +1,144 @@
+import React, { useEffect } from "react";
+import "./App.scss";
+
+let _notes = JSON.parse(localStorage.getItem("notes") || "[]") || [];
```

Comment:
the conditional storage retrieval is not returning an array

Please change to []. NOT "[]"